src/Hng.Application/Features/Blogs/Handlers/GetBlogByIdQueryHandler.cs

```diff
@@ -0,0 +1,17 @@
+﻿using AutoMapper;
+using Hng.Application.Features.Blogs.Dtos;
+using Hng.Application.Features.Blogs.Queries;
+using Hng.Domain.Entities;
+using Hng.Infrastructure.Repository.Interface;
+using MediatR;
+
+namespace Hng.Application.Features.Blogs.Handlers;
+
+public class GetBlogByIdQueryHandler(IRepository<Blog> blogRepository, IMapper mapper) : IRequestHandler<GetBlogByIdQuery, BlogDto>
```

Comment:
Assign constructor params to read only fields eg

>public class GetBlogByIdQueryHandler(IRepository<Blog> blogRepository, IMapper mapper)
{
 private readonly _mapper=mapper;
private readonly _repo=blogRepository;
}